sbncode/CAFMaker/CMakeLists.txt

```diff
@@ -1,89 +1,77 @@
-                  sbnanaobj_StandardRecord
-                  sbnanaobj_StandardRecordFlat
-                  ${ART_FRAMEWORK_CORE}
-                  ${ART_FRAMEWORK_SERVICES_REGISTRY}
-                  ${ART_ROOT_IO_TFILESERVICE_SERVICE}
-                  ${ART_FRAMEWORK_SERVICES}
-                  ${ART_FRAMEWORK_PRINCIPAL}
-                  art_Persistency_Common
-                  art_Utilities canvas
-                  ${MF_MESSAGELOGGER}
-                  ${MF_UTILITIES}
-                  ${FHICLCPP}
-                  cetlib cetlib_except
-                  ${ROOT_EVE_LIB_LIST}
-                  ${ROOT_X3d}
-                  ${ROOT_BASIC_LIB_LIST}
+                  ifdh_art::IFDH_service
+                  sbnanaobj::StandardRecord
+                  sbnanaobj::StandardRecordFlat
+                  art::Framework_Core
+                  art::Framework_Services_Registry
+                  art_root_io::TFileService_service
+                  art::Framework_Principal
+                  art::Persistency_Common
+                  art::Utilities canvas::canvas
+                  cetlib::cetlib cetlib_except::cetlib_except
+                  ROOT::X3d
+                  ROOT::EG
                   caf_RecoUtils
-                  lardataobj_AnalysisBase
-                  lardataobj_RecoBase
-                  larcorealg_Geometry
-                  larcore_Geometry_Geometry_service
-                  larcorealg_GeoAlgo
-                  larsim_MCCheater_BackTrackerService_service
-                  larsim_Utils
-                  nusimdata_SimulationBase
-                  larsim_MCCheater_ParticleInventoryService_service
-                  sbnobj_Common_CRT
-                  sbnobj_Common_Reco
-                  sbnobj_Common_Analysis
-		  lardataalg_DetectorInfo
-                  art_Framework_Services_System_TriggerNamesService_service
-                  sbncode_Metadata_MetadataSBN_service
+                  lardataobj::AnalysisBase
+                  lardataobj::RecoBase
+                  larcorealg::Geometry
+                  larcore::Geometry_Geometry_service
+                  larcorealg::GeoAlgo
+                  larsim::MCCheater_BackTrackerService_service
+                  nusimdata::SimulationBase
+                  larsim::MCCheater_ParticleInventoryService_service
+                  sbnobj::Common_CRT
+                  sbnobj::Common_Reco
+                  sbnobj::Common_Analysis
+                  lardataalg::DetectorInfo
+                  art::Framework_Services_System_TriggerNamesService_service
+                  sbncode::Metadata_MetadataSBN_service
+                  larsim::Utils
                   )
 
-cet_find_library( IFDH NAMES ifdh PATHS ENV IFDHC_LIB NO_DEFAULT_PATH )
-##include_directories( $ENV{IFDHC_FQ_DIR}/inc )
-##include_directories( $ENV{IFDH_ART_FQ_DIR}/inc )
-
-include_directories( $ENV{SRPROXY_INC} )
-
-cet_find_library( IFDH_SERVICE NAMES IFDH_service PATHS ENV IFDH_ART_LIB )
```

Comment:
@chissg Could the removal of this cet_find_library cause the error in the icaruscode build

Reply:
Adding these back did allow the icaruscode build to complete.